c++/src/kj/async-io.c++

```diff
@@ -917,7 +917,7 @@ private:
     Promise<void> writeWithFds(ArrayPtr<const byte> data,
                                ArrayPtr<const ArrayPtr<const byte>> moreData,
                                ArrayPtr<const int> fds) override {
-#if __GNUC__ && !__clang__ && __GNUC__ >= 8
+#if __GNUC__ && !__clang__ && __GNUC__ >= 7
 // GCC 8 decides the open-brace below is "misleadingly indented" as if it were guarded by the `for`
```

Comment:
Could update the comment, too, up to you.

Reply:
Fixed and merged.